.gitattributes

```diff
@@ -1 +1,10 @@
 *.sh eol=lf
+*.md eol=lf
+*.json eol=lf
+*.yaml eol=lf
+.*ignore eol=lf
+*.markdown eol=lf
+nvm-exec eol=lf
+Makefile eol=lf
+bash_completion eol=lf
+
```

Comment:
What do you mean only test files?

There's no reason for any file anywhere to ever have a CRLF, is what I'm saying, and if there's an explicit exception because we need to test CRLFs, we'd add it here.